README.md

```diff
@@ -24,7 +24,20 @@ Generating docs locally can be accomplished by running make gendocs from within
 $ brew tap flyteorg/homebrew-tap
 $ brew install flytectl
 ```
+## Get Started 
 
+### Create a sandbox cluster 
+```bash
+$ docker run --rm --privileged -p 30081:30081 -p 30082:30082 -p 30084:30084 ghcr.io/flyteorg/flyte-sandbox
+```
+
+### Register your first workflow
+
+```bash
+# Run Core workflows
+$ flytectl register files  https://github.com/flyteorg/flytesnacks/release/releases/download/v0.2.89/flytesnacks-core.tgz -d development  -p flytesnacks --archive  
```

Comment:
can we have this download the latest release version?

Reply:
@katrogan that would be hard, as the semantics of the release URL are not know to `flytectl`
IMO @pmahindrakar-oss, @katrogan we should probably have a different cli target
```
flytectl register examples --version v0.2.89
# OR defaults to latest if --version is not provided?
```

Reply:
Also now that we have multiple modules and each with its own archive then we would additionally need option for that too. eg : https://github.com/flyteorg/flytesnacks/releases/download/v0.2.89/flytesnacks-integrations-hive.tgz